vendor/github.com/portworx/sched-ops/k8s/k8s.go

```diff
@@ -91,9 +91,9 @@ type NamespaceOps interface {
 	// GetNamespace returns a namespace object for given name
 	GetNamespace(name string) (*v1.Namespace, error)
 	// CreateNamespace creates a namespace with given name and metadata
-	CreateNamespace(name string, metadata map[string]string) (*v1.Namespace, error)
+	CreateNamespace(string, map[string]string) (*v1.Namespace, error)
```

Comment:
The PR has vendor changes with changes to glide.lock

Make sure you update torpedo vendor using `make vendor` rule and then update this  PR